build.gradle

```diff
@@ -833,6 +833,8 @@ project('test:system') {
         systemProperty "imagePrefix", System.getProperty("imagePrefix", "pravega")  // the default value of imagePrefix is pravega.
         systemProperty "pravegaImageName", System.getProperty("pravegaImageName", "pravega") // pravega image name
         systemProperty "bookkeeperImageName", System.getProperty("bookkeeperImageName", "bookkeeper") // bookkeeper image name
+        systemProperty "zookeeperImageName", System.getProperty("zookeeperImageName", "zookeeper") // zookkeeper image name
+        systemProperty "zookeeperVersion", System.getProperty("zookeeperVersion", "latest") // zookeeper image version
```

Comment:
`zookeeperImageVersion` seems more apt here and eliminates the confusion of it referring the ZK version. Could we change it ?

Reply:
Changed that